Documentation/help_files/medical_dashboard_help.php

```diff
@@ -0,0 +1,235 @@
+                    
+                    <p><?php echo xlt("If the patient is deceased then the deceased notification will appear in red in this section");?>.
+                    
+                    <p><?php echo xlt("For the deceased notification to appear the date of death must be noted under Medical Dashboard > Edit Demographics > Misc");?>.
+                    
+                    <p><?php echo xlt("The help icon will let you access context sensitive help for each of the pages accessed");?>. <i class="fa fa-question-circle fa-lg oe-help-redirect" aria-hidden="true"></i>
+                    
+            </div>
+            </div>
+            <div class= "row" id="section2">
+                <div class="col-sm-12">
+                    <h4 class="oe-help-heading"><?php echo xlt("Nav Bar"); ?><a href="#"><i class="fa fa-arrow-circle-up oe-pull-away oe-help-redirect" aria-hidden="true"></i></a></h4>
+                    <p><?php echo xlt("The Nav Bar allows one to quickly navigate to various parts of the patient's medical record");?>.
+                    
+                    <p><?php echo xlt("The default installation has the following items");?>:
+                    
+                        <ul>
+                            <li><?php echo xlt("Dashboard"); ?></li>
+                            <li><?php echo xlt("History"); ?></li>
+                            <li><?php echo xlt("Report"); ?></li>
+                            <li><?php echo xlt("Documents"); ?></li>
+                            <li><?php echo xlt("Transactions"); ?></li>
+                            <li><?php echo xlt("Issues"); ?></li>
+                            <li><?php echo xlt("Ledger"); ?></li>
+                            <li><?php echo xlt("External Data"); ?></li>
+                        </ul>
+                    
+                    <p><?php echo xlt("Dashboard - summarizes all patient related information");?>.
+                    
+                    <p><?php echo xlt("History - patient's past medical history, family history, personal history");?>.
+                    
+                    <p><?php echo xlt("Report - Generates and downloads the patient's Continuity of Care Record (CCR), Continuity of Care Document (CCD) and Patient Report");?>.
+                    
+                    <p><?php echo xlt("Documents - a repository of the patient's scanned/faxed paper documents. It also the place to download patient specific templates");?>.
+                    
+                    <p><?php echo xlt("Transactions - lists various notes about happenings in a patient's chart with respect to billing, legal, patient request, physician request and also generates a patient referral or counter-referral");?>.
+                    
+                    <p><?php echo xlt("Issues  - summarizes the patient's medical problems, allergies, medications, surgeries and dental issues");?>.
```

Comment:
remove the double space in this constant

Reply:
Done